### DISCUSSION TITLE
global-create-menu-button is always on display in Webkit based browsers

### DISCUSSION THREAD

**Discussion by CleoMenezesJr** [community/community#62013] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

![Screenshot from 2023-07-27 23-37-57](https://github.com/community/community/assets/54215258/c898da67-35dd-4c42-8529-c45e08182c95)

Browser: GNOME Web aka. Epiphany 44.4
Engine: WebKitGTK 2.40.4

**Comment by janxkoci** [comment]
Same issue even in Epiphany 44.6.

Can anybody reproduce it in Safari?

**Comment by mcatanzaro** [comment, marked as answer]
This is fixed by https://bugs.webkit.org/show_bug.cgi?id=253363

**Reply by mcatanzaro** [reply]
See also: https://bugs.webkit.org/show_bug.cgi?id=258290

Will be fixed in WebKitGTK 2.42.0, and also in 2.40.6 if that is ever released. (The current version is 2.40.5 and that might or might not be the last 2.40.)
